frontend/public/components/utils/documentation.tsx

```diff
@@ -1,47 +1,2 @@
-import * as React from 'react';
-import * as _ from 'lodash-es';
-
-import { ExternalLink } from '../utils';
-
 // Prefer the documentation base URL passed as a flag, but fall back to the latest docs if none was specified.
 export const openshiftHelpBase = window.SERVER_FLAGS.documentationBaseURL || 'https://docs.okd.io/latest/';
```

Comment:
Leaving this here instead of moving to const.js because I want to add the enum back for different topics like we had in 3.11.